submissions/description/server_security_misconfiguration/lack_of_security_headers/strict_transport_security/template.md

```diff
@@ -0,0 +1,24 @@
+# Lack of HTTP Strict Transport Security Header
+
+## Overview of the Vulnerability
+
+A lack of HTTP response headers can lead to sensitive data being transmitted unencrypted and being intercepted while in transit.There are multiple HTTP response headers used in communication between the server and client which can be implemented to improve security against well documented vulnerabilities. For example, the HTTP Strict Transport Security (HSTS) header instructs the website to enforce the use of HTTPS.
+
+By not having a HSTS header in place, the application fails to enforce users to connect to it over an encrypted connection. If an attacker is positioned to intercept or modify a user’s network traffic, they can perform a protocol downgrade attack, a Machine-in-the-Middle (MitM) attack, or hijack a user’s cookies. This aids an attacker in gaining access to the data being sent between the server and client, such as login credentials or application cookies.
```

Comment:
Mild direct improvements for this templates overview

A lack of the HTTP response header for Strict Transport Security can lead to sensitive data being transmitted unencrypted and being intercepted while in transit. There are multiple HTTP response headers used in communication between the server and client which can be implemented to improve security against well documented vulnerabilities. For example, the HTTP Strict Transport Security (HSTS) header instructs the website to enforce the use of HTTPS.

By not having a HSTS header in place, the application fails to enforce users to connect to it over an encrypted connection. If an attacker is positioned to intercept or modify a user’s network traffic, they can perform a protocol downgrade attack, a Persons-in-the-Middle (PitM) attack, or hijack a user’s cookies. This aids an attacker in gaining access to the data being sent between the server and client, including the login credentials or application cookies.

Reply:
Cheers for the feedback, please see latest commit